cargo-dist/src/config.rs

```diff
@@ -1530,6 +1530,7 @@ pub(crate) fn parse_metadata_table_or_manifest(
             let src = SourceFile::load_local(manifest_path)?;
             parse_generic_config(src)
         }
+        WorkspaceKind::Javascript => todo!(),
```

Comment:
Need to fix this - it looks like the `path` reference is ignoring the features we asked for, so we suddenly got asked about this.

Reply:
whaaat

Reply:
oh